app/guest/[id]/components/GuestBgm.tsx

```
@@ -0,0 +1,110 @@
+  }, [isOn, src]);
+
+  useEffect(() => {
+    const audio = audioRef.current;
+    return () => {
+      audio?.pause();
+    };
+  }, []);
+
+  return (
+    <>
+      <audio ref={audioRef} preload="none" />
```

Biome (2.3.14)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)